logstash/tests/logstash_test.py

```diff
@@ -876,3 +876,33 @@ def test_setting_fullnameOverride():
         ]
         == "logstash"
     )
+
```

Comment:
An empty line is missing here to make [Black](https://black.readthedocs.io/en/stable/) formatter happy in [elastic+helm-charts+pull-request+lint-python/484](https://devops-ci.elastic.co/job/elastic+helm-charts+pull-request+lint-python/484/label=docker&&virtual&&ubuntu-18.04/console).

Can you run `black <CHART>/tests/<CHART>_test.py`?

Reply:
Done.